hibike/hibike_process.py

```diff
@@ -126,6 +130,11 @@ def device_read_thread(index, ser, instructionQueue, errorQueue, stateQueue):
                 if uid is not None:
                     instructionQueue.put(("heartResp", [uid]))
 
+def batch_data(data, stateQueue):
+    while True:
+        time.sleep(.04)
```

Comment:
This is a hardcoded constant hiding here. Moreover, the value of this constant has impact on performance.

It might be good to have a comment explaining why batch reads are required, and some brief tips on how this constant value was determined.